int64 xmin
int64 ymin
int64 xmax
int64 ymax
int16 id
string Class
bool lost_flag 